diagnostic.md

```diff
@@ -156,9 +172,10 @@ div {
 }
 ```
 
-<!-- your answer starts here -->
+divTwo, divThree, and divFive will be arranged on the same horizontal plane on
+the left side, while divOne will be floated to the right of the other divs (also on the same line).
+divFour will appear right after the line break, underneath the div's.   
```

Comment:
Order:
2,3,1
4,5

Div 4 clears the old float, thus bringing it to the next line, and is floated again. From this, div5 floats next to it.